ansible_wisdom/main/settings/base.py

```diff
@@ -436,6 +437,50 @@
     ),
 )
 
+DUMMY_MODEL_LLM_RESPONSE_DICT = os.environ.get(
```

Comment:
Could you move this inside the dummy_client.py driver? This will also help to avoid the json.dumps() call.

I'm not totally sure we need to be able to overwrite this content. If it happens to be the case, I feel like it would be more elegant to have a configuration key pointing on a directory with some sample files.

Reply:
Updated the code in the latest commit. I added that because `DUMMY_MODEL_RESPONSE_BODY` was there, but I suspect that was not used either....